src/components/list/dragSort.js

```diff
@@ -88,7 +89,7 @@ class DragSort extends Component {
     return connectDragPreview(
       connectDropTarget(
         <div
-          className="intellyo-drag-sort"
+          className={ classnames('intellyo-drag-sort', { 'is-dragging': this.props.isDragging }) }
```

Comment:
`is-dragging` is a state modifier and too generic since css is global (at least how we use it). We write this in this way: `intellyo-drag-sort--is-dragging` to avoid naming conflicts in the global css namespace.